packages/nectar/src/sagas/publicChannels/publicChannels.adapter.ts

```diff
@@ -7,7 +7,7 @@ export const communityChannelsAdapter =
   createEntityAdapter<CommunityChannels>()
 
 export const publicChannelsAdapter = createEntityAdapter<PublicChannel>({
-  selectId: (channel) => channel.name
+  selectId: (channel) => channel.address
```

Comment:
Entity adapters needs to identify the entries by some id. By default it's just the model's `id` field, but in this case there's no such field in the channel schema so we need to point the adapter to something else. 
At some point I managed to add malformed channel data to the database and after replication it contained only the `address` field (is it the key in database?) so to prevent frontend errors I secured it by setting `address` instead of the `name` as the entry id in the adapter

Reply:
this seems related to the other question of "what's an identifier for channel we know is unique?" 

it seems like we should figure that out and use that here. or am i misunderstanding?

Reply:
marek suggests using `ref`